core/src/test/java/io/reacted/core/reactorsystem/ServiceTest.java

```diff
@@ -29,15 +29,15 @@
     @BeforeEach
     void setUp() {
         ReActorSystemConfig reActorSystemConfig = ReActorSystemConfig.newBuilder()
-                                                                     .setReactorSystemName(CoreConstants.REACTED_ACTOR_SYSTEM)
-                                                                     .setMsgFanOutPoolSize(2)
-                                                                     .setLocalDriver(SystemLocalDrivers.DIRECT_COMMUNICATION)
-                                                                     .addDispatcherConfig(DispatcherConfig.newBuilder()
-                                                                                                          .setDispatcherName("TestDispatcher")
-                                                                                                          .setBatchSize(1_000)
-                                                                                                          .setDispatcherThreadsNum(1)
-                                                                                                          .build())
-                                                                     .build();
+                .setReactorSystemName(CoreConstants.REACTED_ACTOR_SYSTEM)
+                .setMsgFanOutPoolSize(2)
+                .setLocalDriver(SystemLocalDrivers.DIRECT_COMMUNICATION)
+                .addDispatcherConfig(DispatcherConfig.newBuilder()
```

Comment:
why do you create an extra dispatcher? I don't see it used anywhere

Reply:
removed